File: openapi/code_samples/go/utilitypayments/accounts_balance/code.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {

	url := "https://utilities.reloadly.com/accounts/balance"
	method := "GET"

	client := &http.Client{}
	req, err := http.NewRequest(method, url, nil)

	if err != nil {
		fmt.Println(err)
		return
	}
	req.Header.Add("Accept", "application/com.reloadly.utilities-v1+json")
	req.Header.Add("Authorization", "Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")

	res, err := client.Do(req)
	if err != nil {
		fmt.Println(err)
		return
	}
	defer res.Body.Close()

	body, err := ioutil.ReadAll(res.Body)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(string(body))
}

File: openapi/code_samples/csharp/utilitypayments/accounts_balance/code.cs
using System;
using System.Threading.Tasks;
using System.Net.Http;
using System.Text;
using Newtonsoft.Json;

namespace WebAPIClient {
  class Program {
    static async Task Main(string[] args) {
      await ApiCall();
    }
    
    private static async Task ApiCall() {
      var message = new HttpRequestMessage(HttpMethod.Get, "https://utilities.reloadly.com/accounts/balance");
      message.Headers.TryAddWithoutValidation("Authorization", "Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
      message.Headers.TryAddWithoutValidation("Accept", "application/com.reloadly.utilities-v1+json");
      var httpClient = new HttpClient();
      var response = await httpClient.SendAsync(message);
      var responseBody = await response.Content.ReadAsStringAsync();
      var result = JsonConvert.DeserializeObject < dynamic > (responseBody);
      Console.WriteLine(result);
    }
  }
}